README.md

```diff
@@ -16,8 +16,9 @@ The easiest way to get up and running is to use our pre-configured Docker Compos
   
   1. Install Docker Compose and ensure Docker is running. This should be included with the [Docker Desktop](https://www.docker.com/products/docker-desktop/) installation
   2. Run `curl -L https://ghst.ly/getbhce | docker compose -f - up`
-  3. Locate the randomly generated password in the terminal output of Docker Compose
```

Comment:
Looks like numbering got goofed in the changes. If you wouldn't mind confirming and then it looks good!